CMakeLists.txt

```diff
@@ -3,6 +3,21 @@ cmake_minimum_required(VERSION 2.8)
 
 set(Ultrasound_LIBRARIES Ultrasound)
 
+if(NOT ITK_SOURCE_DIR)
+  find_package(ITK REQUIRED)
+  list(APPEND CMAKE_MODULE_PATH ${ITK_CMAKE_DIR})
+endif()
+
+option(ITKUltrasound_USE_clFFT "Build ITKUltrasound 1D FFT filters with clFFT as the back-end.\n\
+  https://github.com/clMathLibraries/clFFT" ${ITK_USE_GPU})
+mark_as_advanced(ITKUltrasound_USE_clFFT)
+if(ITKUltrasound_USE_clFFT)
+  find_package(clFFT REQUIRED)
+  list(APPEND Ultrasound_LIBRARIES ${CLFFT_LIBRARIES})
+  set_property(DIRECTORY APPEND PROPERTY INCLUDE_DIRECTORIES ${CLFFT_INCLUDE_DIRS})
+  set_property(DIRECTORY APPEND PROPERTY COMPILE_DEFINITIONS ITKUltrasound_USE_clFFT)
```

Comment:
Instead, can we add these to the include directories and compile definitions for the module?

Reply:
How to do it for compile definitions? This is done for VTK: `set_property(DIRECTORY APPEND PROPERTY COMPILE_DEFINITIONS ${Ultrasound_VTK_DEFINITIONS})`

I assume include dirs are done like this: `set(Ultrasound_SYSTEM_INCLUDE_DIRS ${Ultrasound_VTK_INCLUDE_DIRS})`?

Reply:
> How to do it for compile definitions? This is done for VTK: set_property(DIRECTORY APPEND PROPERTY COMPILE_DEFINITIONS ${Ultrasound_VTK_DEFINITIONS})

It could be done this way or with `configure_file`. If it is done this way, then `set_property(DIRECTORY [...]` also needs to be added to the module `EXPORT_CODE` (ITKVTKGlue does this).

> I assume include dirs are done like this: set(Ultrasound_SYSTEM_INCLUDE_DIRS ${Ultrasound_VTK_INCLUDE_DIRS})?

Yes.